modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc

```diff
@@ -6,22 +6,22 @@
 [id="reviewing-cluster-status-from-the-openshift-cluster-manager_{context}"]
 = Reviewing the cluster status from {cluster-manager-first}
 
-You can review detailed information about the status of your cluster in the {cluster-manager}.
+From the {product-title} web console, you can review detailed information about the status of your cluster on {cluster-manager}.
 
 .Prerequisites
 
 * You have access to the cluster as a user with the `cluster-admin` role.
 
 .Procedure
 
-. In the *Administrator* perspective, navigate to *Home* -> *Overview* -> *Details* -> *{cluster-manager}* to open the *Overview* page for the cluster in {cluster-manager-url}.
+. In the *Administrator* perspective, navigate to *Home* -> *Overview* -> *Details* -> *Cluster ID* -> *{cluster-manager}* to open the *Clusters* page for the cluster in {cluster-manager-url}.
```

Comment:
A few small issues -

1. Does step 1 take you directly into the details for a cluster (ie. this view https://console.redhat.com/openshift/details/s/1U9JqcRiAXsrylGHJAGHJGHGHvS6#overview) , or the general list of all the clusters (this page https://console.redhat.com/openshift) ? If the navigation path mentioned takes you directly into a single cluster's Overview page, I'd call it 'the cluster details page' or 'the cluster Overview page' - we use wording like that in the OCM guide - https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2022/html-single/managing_clusters/index#assembly-managing-clusters

2. I know this isn't part of this PR, but the attribute {cluster-manager-url} needs to be fixed as this is rendering as an incorrect product/service name. 

I'd suggest writing it as "in the Hybrid Cloud Console" with same URL for the hyperlink linked to the words "Hybrid Cloud Console". I think it's more important to emphasize the Hybrid Cloud Console (and the name OpenShift Cluster Manager might change or disappear in the upcoming changes to the console.)

@aldiazRH - please provide a sanity check for me :)

Reply:
Thanks, @dayleparker .

1. I see a Clusters page that lists all the clusters, so I won't change the name in this instance. Unless @aldiazRH , the production environment is different?

2. Sadly, 3266 references to the cluster-manager-url manager exist. If I change this attribute, I would need to spend a lot of time checking all instances read correctly. If the OCM Hybrid Cloud Console name were to change, the justification for doing a wider sweep would be easier. 

@sjstout , what do you think? Keep `:cluster-manager-url: link:https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console]` the same in the `common-attributes.adoc` file? I do agree that the link name does not directly align with what displays on the Hybrid Web Console.

Reply:
@dfitzmau 

1. I see the same, so I'm happy to leave it as is. 

2. I agree with the problematic naming, but I also agree that the change for that is out of scope for this specific PR. That would have to be a separate issue, which I could take on, as I'm the one working on OCM docs.

Reply:
Thanks, @aldiazRH . Just wondering if the following note should now be removed (based on my proposed update)?

![Screenshot from 2023-02-02 17-00-48](https://user-images.githubusercontent.com/57954076/216391988-0f5e2e29-ab2b-4772-a0c0-acfb8b3b6611.png)

Thanks for taking on part 2!

Would you happen to know an engineer that could review my proposed update?

Reply:
Yes, I agree, the note is no longer needed in this situation. :+1: 

As for eng, I would reach out to Dave Taylor or Kim Doberstein to see if they, or someone on their team can take a look. That'd be my first guess, anyway.